DESCRIPTION

```diff
@@ -82,6 +82,7 @@ URL: https://rstudio.github.io/bslib/, https://github.com/rstudio/bslib
 BugReports: https://github.com/rstudio/bslib/issues
 Config/testthat/edition: 3
 Config/Needs/routine:
+    chromote,
```

Comment:
Is this needed?

Reply:
Yup! The idea is that routine will call `devtools::document()` which will result in up-to-date screenshots of the examples. That process requires `chromote` for the screenshotting.